### PR TITLE
Fixes for webpack

### PR DESCRIPTION
**Pull request by hdf1996**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import React from'react';
-import ReactDom from'react-dom';
+import React from 'react';
+import ReactDOM from 'react-dom';
```
Comment (hdf1996) [inline src/index.js:2]: After the fixes of eslint you had a typo here

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 import Header from './components/Header';
 // import Header from './components/Header';
 
 
-ReactDOM.render(<Header />, document.getElementById('app'));
+ReactDOM.render(<Header />, document.getElementById('root'));
```
Comment (hdf1996) [inline src/index.js:7]: The element you are injecting react was called `root`, not `app`

File: webpack.config.js [hunk 3]
```diff
@@ -18,12 +18,9 @@ module.exports = {
         test: /\.(js|jsx)$/,
         exclude: /node_modules/,
         use: {
-          loader: 'babel-loader',
-          options: {
-            presets: ['es2015', 'react']
-          },
+          loader: 'babel-loader'
```
Comment (hdf1996) [inline webpack.config.js:21]: At the end, with a good `.babelrc` we were able to remote the options here

File: .babelrc [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env", "@babel/preset-react"]
```
Comment (hdf1996) [inline .babelrc:2]: Here we could use the presets we had talked about, but tried following `@babel/**` pattern to match your guide

**Review by ferchuqui — APPROVED** [review]

**Pull request merged**